### ISSUE TITLE
Demo record and short sentence for Gazetteer

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-eXist-app#701]
@tacarlson re issue https://github.com/srophe/srophe-eXist-app/issues/695 we need to generate a short description of the gaz and a sample record. Would you weigh in on these:

Record 78

Short description:  "The Syriac Gazetteer is a geographical reference work covering more than 2,000 places relevant to Syriac studies."

**Comment by tacarlson** [comment]
Sample record should definitely be place/78


As for the short description, I'm not too keen on including the number of entries (it will change, it sounds small to some people, etc.).  How does the following sound (essentially boiled down from the about page)?


"The Syriac Gazetteer is a geographical reference work of places relevant to Syriac Studies, including places named in Syriac texts and places where scholarship on Syriac is being produced. There are no temporal or spatial restrictions for the geographic database, which collects places relevant to every period of Syriac studies."


If that is too wordy (I don't know what the parameters are), then perhaps the following:


"The Syriac Gazetteer is a geographical reference work of all places relevant to Syriac Studies, with no temporal or spatial restrictions."


What do you think?

________________________________

**Comment by wsalesky** [comment]
@tacarlson Thanks! I used the shorter one. See:
http://wwwb.library.vanderbilt.edu/exist/apps/srophe/index.html

It will popup when you role over the image under 'Places' (I also noticed that it sits on top of the arrow and is hard to read, I will try to give the carousel margins on far left and right for the arrows, so they do not obstruct any text or images.)

**Comment by jsaintlauren** [comment]
1.Sample record for Qadishe:

Record 1102:
http://wwwb.library.vanderbilt.edu/exist/apps/srophe/person/1102

Description: *"**Qadishe *is a digital catalogue of saints venerated in the
Syriac tradition."

2. Sample Record for BHSE: work 694

http://wwwb.library.vanderbilt.edu/work/694

The *Bibliotheca Hagiographica Syriaca Electronica *is a database of Syriac
hagiographic literature.
Thanks, Winona!
